src/validation/validators/rules.js

```diff
@@ -0,0 +1,44 @@
+import { check } from 'express-validator';
+
+// add validation rules here.
+
+/* regex description
+the regex /^[\w'\-,.][^0-9_!¡?÷?¿/\\+=@#$%ˆ&*(){}|~<>;:[\]]{1,250}$/
+is made up of two character set, a valid set and a negated set found between
+[], with a quantifier {1,250}
+
+the first character set contains characters that should be allowed in the match,
+[\w'\-,.] => any "alphanumeric and underscores", any "'-,."
+the second character set is a negated set, matches any character not in the set
+this prevent us from matching numbers, $@*()+= etc
+the last {1,250} is a quantifier specifying that the character been matched
+should be > 1 and <= 251
+
+the ^ and $ runs the match from the beginning and end of the string
+*/
+export const userRegister = [
+  check('firstName')
+    .matches(/^[\w'\-,.][^0-9_!¡?÷?¿/\\+=@#$%ˆ&*(){}|~<>;:[\]]{1,250}$/)
+    .withMessage(
```

Comment:
`[\w'\-,.] => any "alphanumeric and underscores", any "'-,."`. The only special characters that may be found in a name is `' or -`. Also, I have not seen where a name is made up of `alphanumeric` characters or `underscores`. 

I know you can have a name like `Ma'John` or `John-Sleg`. However, I have never seen a name like `Ma4John` or `Ma_John`. Hence the regex needs improvement.

Since the same regex is used multiple times, you may need to assign it to a variable and then use it when needed.